docs/detections/api/rules/signals-api-overview.asciidoc

```diff
@@ -245,3 +250,107 @@ Example response:
   "failures": []
 }
 --------------------------------------------------
+
+
+==== Apply alert tags
+
+Add and remove alerts tags on alerts.
+
+===== Request URL
+
+`POST <kibana host>:<port>/api/detection_engine/signals/tags`
+
+===== Request body
+
+A JSON object with the `tags` and `_id` fields:
```

Comment:
```suggestion
A JSON object with the `tags` and `ids` fields:
```
Looks like `_id` is the field returned in the response, but the request field is `ids`.